src/extensions/scratch3_playspot/index.js

```diff
@@ -992,43 +944,18 @@ class Scratch3PlayspotBlocks {
                     blockType: BlockType.BOOLEAN,
                     arguments: {
                         SATELLITES: {
-                            type: ArgumentType.REPORTER
-                        }
-                    }
-                },
-                '---',
-                {
-                    opcode: 'whenGameModeEquals',
-                    text: 'When game mode changes to: [MODE]',
-                    blockType: BlockType.HAT,
-                    arguments: {
-                        MODE: {
                             type: ArgumentType.STRING,
-                            menu: 'modes',
-                            defaultValue: 'Unknown'
-                        }
-                    }
-                }, {
-                    opcode: 'isGameModeEqual',
-                    text: 'When the game mode equals: [MODE]',
-                    blockType: BlockType.BOOLEAN,
-                    arguments: {
-                        MODE: {
-                            type: ArgumentType.STRING,
-                            menu: 'modes',
-                            defaultValue: 'Unknown'
+                            defaultValue: defaultSatellite
                         }
                     }
                 },
-                '---',
                 {
-                    opcode: 'displayLightSequence',
+                    opcode: 'displayTheLightSequence',
```

Comment:
Not sure I understand This name change...

Reply:
I can change it back if need be. I was testing something where I created a new block that copied that one and the function to follow and didn't change it back. The corresponding function has the THE inserted as well so everything still works. Just didn't change it back to the original function name without the THE.

Reply:
Would prefer if you did, please.

Reply:
Fixed